adhocracy-plus/config/settings/base.py

```diff
@@ -560,3 +551,134 @@
 CELERY_RESULT_BACKEND = "redis://localhost:6379"
 CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
 CELERY_RESULT_EXTENDED = True
+
+# CKEditor config
+CKEDITOR_UPLOAD_PATH = "uploads/"  # to be removed after django upgrade and ckeditor_uploader removal
+CKEDITOR_5_FILE_STORAGE = "adhocracy4.ckeditor.storage.CustomStorage"
+CKEDITOR_5_PATH_FROM_USERNAME = True
+CKEDITOR_5_ALLOW_ALL_FILE_TYPES = True
+CKEDITOR_5_UPLOAD_FILE_TYPES = ["jpg", "jpeg", "png", "gif", "pdf", "docx"]
+CKEDITOR_5_CONFIGS = {
+    "default": {
+        "language": "de",
+        "toolbar": [
+            "bold",
+            "italic",
+            "underline",
+            "|",
+            "link",
+            "bulletedList",
+            "numberedList",
+        ],
+        "list": {
+            "properties": {
+                "styles": "true",
+                "startIndex": "true",
+                "reversed": "true",
+            }
+        },
+        "link": {"defaultProtocol": "https://"},
+    },
+    "image-editor": {
+        "language": "de",
```

Comment:
we need to keep in mind that we somehow need to fix the language switching, might not have to be in this PR but before we release

Reply:
upstream django-ckeditor-5 accepted the language PR, so this should be fine now